src/kubernetes.cc

```diff
@@ -792,30 +792,39 @@ void KubernetesReader::WatchMaster(
   http::client::request request(endpoint);
   request << boost::network::header(
       "Authorization", "Bearer " + KubernetesApiToken());
-  const bool verbose = config_.VerboseLogging();
-  if (verbose) {
-    LOG(INFO) << "WatchMaster(" << name << "): Contacting " << endpoint;
-  }
   try {
+    const bool verbose = config_.VerboseLogging();
     if (verbose) {
-      LOG(INFO) << "Locking completion mutex";
+      LOG(INFO) << "WatchMaster(" << name << "): Will retry "
```

Comment:
I'm not sure I'm happy about this logging of retries upfront. Would rather be silent on the first try and notify of retries as they happen, e.g.:
```c++
    for (int i = 0; i < config_.KubernetesUpdaterWatchRetries(); ++i) {
      if (verbose) {
        if (i != 0) {
          LOG(INFO) << "WatchMaster(" << name << "): Retrying; attempt #" << i
                    << " of " << config_.KubernetesUpdaterWatchRetries();
        }
        LOG(INFO) << "WatchMaster(" << name << "): Contacting " << endpoint;
      }
      ...
    }
```

Reply:
This `LOG(INFO)` before the loop is now superfluous.